File: ch4 supplemental/repl2.cs
using System;
class MainClass {
  public static void Main (string[] args) {
    Console.WriteLine("Do you have children? (Y/N)");
    string hasChildren = Console.ReadLine();
    
    if(hasChildren == "Y" || hasChildren=="y"){
    
    	Console.WriteLine("How many children?");
    	string childCount = Console.ReadLine();
    	Console.WriteLine("You have " + childCount + " children.");
    }
    
    Console.WriteLine("Thanks for your participation, questionaire complete.");
    }
  } 
      

File: ch4 supplemental/repl2.js
var hasChildren = prompt("Do you have children (Y/N)");

if(hasChildren == "Y" || hasChildren=="y") 
{
	var childCount = prompt("How many children?");
	
	if(parseInt(childCount)!=NaN && isFinite(childCount)){
	console.log("You have " + childCount + " children");
	
	
	}
	
}

console.log("Thanks for your participation, questionnaire complete.")